File: app/src/main/java/top/androidman/superbutton/MainActivity.java
package top.androidman.superbutton;

import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        getSupportActionBar().hide();
        setContentView(R.layout.activity_main);
    }

    public void test(View view) {
        Toast.makeText(this, "是不是很神奇", Toast.LENGTH_SHORT).show();
    }
}

File: app/src/main/java/top/androidman/superbutton/MainActivity.kt
package top.androidman.superbutton

import android.os.Bundle
import android.view.View
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        supportActionBar!!.hide()
        setContentView(R.layout.activity_main)

    }

    fun test(view: View?) {
        Toast.makeText(this, "是不是很神奇", Toast.LENGTH_SHORT).show()
    }
}